roles/bmc_fw_update/vars/main.yml

```diff
@@ -1,2 +1,7 @@
 ---
 # vars file for bmc_fw_update
+
+firmware:
+  url: https://content.mellanox.com/BlueField/BMC/23.10-1-oct-2023
+  bmc: bf3-bmc-23.10-5_opn.fwpkg
+  cec: cec1736-ecfw-00.02.0152.0000-n02-rel-prod.fwpkg
```

Comment:
I don't think it is used currently, best to have it in a separate role

Reply:
sure